strawberryfields/utils.py

```diff
@@ -583,21 +597,25 @@ def random_symplectic(N, passive=False):
     return O @ Sq @ P
 
 
-def random_interferometer(N):
+def random_interferometer(N, real=False):
     r"""Random unitary matrix representing an interferometer.
 
     For more details, see :cite:`mezzadri2006`.
 
     Args:
         N (int): number of modes
+        real (bool): return a random real orthogonal matrix
 
     Returns:
         array: random :math:`N\times N` unitary distributed with the Haar measure
     """
-    z = randnc(N, N)/np.sqrt(2.0)
+    if real:
+        z = randn(N, N)
+    else:
+        z = randnc(N, N) / np.sqrt(2.0)
```

Comment:
curious why the random complex numbers are normalized but the real ones are not? :thinking:

Reply:
@nquesada how come the complex numbers are normalized differently than real?